index.html

```diff
@@ -87,10 +87,10 @@ <h2>We welcome contributors!</h2>
             <div class="container">
                 <div class="row">
                     <div class="col-lg-6">
-                    {% include news.html caption=true title=true limit=3 %}
+                    {% include news.html caption=true title=true limit=3 caption_url="/about/news" %}
                     </div>
                     <div class="col-lg-6 mt-3 mt-lg-0">
-                    {% include events.html caption=true title=true event_type="upcoming_event" %}
+                    {% include events.html caption=true title=true event_type="upcoming_event" caption_url="/about/events" %}
```

Comment:
Does this remove the "upcoming event" portion of the landing page? Might be best to not have it there as it would be empty.

Reply:
yes, since there are no upcoming events, the whole block will not be shown ;)